integration/tests/command_line/deploy_env_without_mod_update.rb

```diff
@@ -0,0 +1,71 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-90 - C62419 - Deploy Environment without Module Update'
+
+#Init
+master_certname = on(master, puppet('config', 'print', 'certname')).stdout.rstrip
+environment_path = on(master, puppet('config', 'print', 'environmentpath')).stdout.rstrip
+prod_env_path = File.join(environment_path, 'production')
```

Comment:
Add a blank new-line after this statement to match the formatting of the other tests.